.travis.yml

```diff
@@ -19,3 +19,4 @@ script:
   - node --version
   - npm --version
   - npm test
```

Comment:
It might have been better to replace this one with `npm run test:coverage`.

`npm test` and `npm run test:coverage` will run the tests twice and generated coverage. 

Think of it like this
```
npm test
npm run test:coverage
=
test and test => coverage
```

```
npm run test:coverage
=
test => coverage
```